content/api/commands/screenshot.md

```diff
@@ -185,27 +187,36 @@ This behavior can be changed by passing the `{overwrite: true}` option to
 
 - For a failure screenshot, the default naming scheme is used and the name is
   appended with ` (failed)`:
-  `{screenshotsFolder}/{specPath}/{testName} (failed).png`
+  `{screenshotsFolder}/{specPathStrippedOfCommonAncestors}/{testName} (failed).png`
 
 For example, given a spec file located at `cypress/e2e/users/login_spec.js`:
 
 ```javascript
 describe('my tests', () => {
   it('takes a screenshot', () => {
-    cy.screenshot() // cypress/screenshots/users/login_spec.js/my tests -- takes a screenshot.png
-    cy.screenshot() // cypress/screenshots/users/login_spec.js/my tests -- takes a screenshot (1).png
-    cy.screenshot() // cypress/screenshots/users/login_spec.js/my tests -- takes a screenshot (2).png
+    // NOTE: Because this file has multiple screenshots and therefore each screenshot has a common ancestor path of `/users/` and therefore stripped from the path.
```

Comment:
I'm having trouble parsing this sentence. Can you maybe split it into two sentences?

Reply:
I separated the `cy.screenshot()` and the comments into separate lines. Do you find this more readable. Changes can be found here https://github.com/cypress-io/cypress-documentation/pull/4326/commits/627caf0088f3f144dc47eeecf0e5ea6501e1e482